BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -289,8 +290,8 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     
 
     func updateDeadbeatVisibility() {
-        guard let user = currentUserManager.user(context: viewContext) else { return }
-        self.deadbeatView.isHidden = !user.deadbeat
+        let isKnownDeadbeat = currentUserManager.user(context: viewContext)?.deadbeat == true
```

Comment:
```suggestion
        let isKnownDeadbeat = currentUserManager.user(context: viewContext)?.deadbeat ?? false
```
This is the same, right?

Reply:
Effectively, yes